README.md

```diff
@@ -14,9 +14,13 @@
 Before you begin, make sure you have all the necessary libraries installed :
 
 ```bash
-pip install git+https://github.com/huggingface/optimum-onnx.git
+git clone https://github.com/huggingface/optimum-onnx.git
+cd optimum-onnx
+pip install .
 ```
 
+⚠️   Optimum ONNX cannot be installed in editable mode for now.
```

Comment:
it actually can be installed in editable mode (with static optimum), you just can't install both optimum and optimum-onnx in editable mode at the same time.

Reply:
not sure how optimum-onnx can be installed in editable mode with static optimum, would be cool for you to open a PR to update the installations instructions (the ones from this PR is a bit confusing so closing)